### ISSUE TITLE
Application of algorithm

### ISSUE THREAD

**Issue by nuo-1** [yewzijian/RegTR#9]
Is this algorithm applicable to outdoor urban scenes?

**Comment by yewzijian** [comment]
Yes, it does not make assumptions on the scene type. You might need to tune the kpconv parameters to better handle large scenes though.

**Comment by nuo-1** [comment]
> Yes, it does not make assumptions on the scene type. You might need to tune the kpconv parameters to better handle large scenes though.

Thanks for your reply. I think I know what to do next.
